miden-tx/src/tests/kernel_tests/test_prologue.rs

```diff
@@ -17,50 +17,39 @@ use miden_lib::transaction::{
     TransactionKernel,
 };
 use miden_objects::{
+    accounts::account_id::testing::ACCOUNT_ID_REGULAR_ACCOUNT_UPDATABLE_CODE_OFF_CHAIN,
     assembly::ProgramAst,
     testing::{
-        account::MockAccountType,
-        notes::AssetPreservationStatus,
+        constants::FUNGIBLE_FAUCET_INITIAL_BALANCE,
         storage::{generate_account_seed, AccountSeedType},
     },
-    transaction::{PreparedTransaction, TransactionArgs, TransactionScript},
+    transaction::{TransactionArgs, TransactionScript},
     Digest, FieldElement,
 };
-use vm_processor::AdviceInputs;
+use vm_processor::{AdviceInputs, ONE};
 
-use super::{build_module_path, ContextId, Felt, Process, ProcessState, Word, TX_KERNEL_DIR, ZERO};
-use crate::testing::{utils::consumed_note_data_ptr, MockHost, TransactionContextBuilder};
-
-const PROLOGUE_FILE: &str = "prologue.masm";
-
-/// Loads epilogue file and returns the complete code formatted as
-/// "{prologue_code}{code}"`
-#[cfg(feature = "std")]
-fn insert_prologue(code: &str) -> String {
```

Comment:
This too wasn't really necessary, and using `use` statements was sufficient for the tests.